cmd/mkpkg/main.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// mkpkg builds the tailscale rpm and deb packages.
```

Comment:
Tailscale?

Reply:
Fixed. I meant specifically a package called "tailscale" at the time because it hardcoded stuff, but now it's generic enough to package e.g. tailcontrol someday.